css/_components.scss

```diff
@@ -51,3 +51,69 @@
         }
     }
 }
+
+
+
+/**
+ * The background-video mixin styles an edge-to-edge background video. Because 
+ * elements within .hero are absolutely positioned, .hero’s height gets wiped 
+ * out. We assign it a faux-height via a ::before pseudo element.
+ *
+ * Emmet: .background-video>.hero>.panel{$1}+video
+ */
+@mixin background-video() {
+    .container {
```

Comment:
I'm not sure why you're defining .container here; I don't think you need/want this selector or block.